src/ui/handler/box_zoom.js

```diff
@@ -147,13 +147,20 @@ class BoxZoomHandler {
         }
     }
 
-    _finish() {
-        this._active = false;
-
+    _unbind() {
         window.document.removeEventListener('mousemove', this._onMouseMove, false);
         window.document.removeEventListener('keydown', this._onKeyDown, false);
         window.document.removeEventListener('mouseup', this._onMouseUp, false);
+    }
+
+    teardown() {
```

Comment:
In the rest of the codebase, we use `remove()` for this action. Let's keep up the consistency.